src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -29,9 +29,24 @@ export const highSchoolStatusLabels = {
 };
 
 export const benefitsRelinquishedWarning = (
-  <div className="usa-alert usa-alert-warning usa-content edu-warning-single-line">
+  <div className="usa-alert usa-alert-warning usa-content">
     <div className="usa-alert-body">
-      I certify that I understand the effects that this election to receive DEA or FRY scholarship benefits will have on my eligibility for DIC payments, and I elect to receive the selected scholarship benefit on the following date:
+      I certify that I understand the effects that this election to receive DEA or FRY scholarship benefits will have on my eligibility for DIC payments, and I elect to receive the selected scholarship benefit on the above date.
     </div>
   </div>
 );
+
+export const benefitsDisclaimerSpouse = (
```

Comment:
Small nit, but the below is written as a function and this is just a const. Perhaps make them consistent?